build.gradle

```diff
@@ -439,8 +441,12 @@ apply from: 'https://raw.githubusercontent.com/FWDekker/intellij-randomness/mast
 
 runPluginVerifier {
   pluginFileName = "${intellij.pluginName}-${version}"
-  ides = [ 'IC-2019.3', 'IC-2020.1', 'IC-2020.1.2' ]
   verifierVersion = '1.241'
+  ides = [
+    intellijVersions.earliestSupported,
+    *intellijVersions.inBetweenForCompatibilityChecks,
```

Comment:
What does this asterisk mean?

Reply:
Expand the list

Reply:
So `[a, *b, c]` is pretty much `[a] + b + [c]`